tests/devapps/NetFxConsoleTestApp/Program.cs

```diff
@@ -20,6 +20,22 @@ namespace NetFx
 {
     public class Program
     {
+        private const string Claims = @" {
+   ""userinfo"":
```

Comment:
@bgavrilMS : is this the way customers can today request claims? (without a specific API) ?

Reply:
Yes, that's what the OIDC spec states, and I'm pretty sure AAD does the right thing. https://openid.net/specs/openid-connect-core-1_0.html#ClaimsParameter 
```json
{
   "userinfo":
    {
     "given_name": {"essential": true},
     "nickname": null,
     "email": {"essential": true},
     "email_verified": {"essential": true},
     "picture": null,
     "http://example.info/claims/groups": null
    },
   "id_token":
    {
     "auth_time": {"essential": true},
     "acr": {"values": ["urn:mace:incommon:iap:silver"] }
    }
  }
```

And Client Capabilities are a special claim for the access token:

```json
{
   "userinfo":
    {
     "given_name": {"essential": true},
     "nickname": null,
     "email": {"essential": true},
     "email_verified": {"essential": true},
     "picture": null,
     "http://example.info/claims/groups": null
    },
   "id_token":
    {
     "auth_time": {"essential": true},
     "acr": {"values": ["urn:mace:incommon:iap:silver"] }
    },
    "access_token":
    { 
      "xms_cc": { "values": ["llt"] }
    }
  }
```